tightvnc.sls

```diff
@@ -1,15 +1,22 @@
 tightvnc:
-  '2.7.10.0':
+  {% for version in ['2.8.11', '2.8.8', '2.8.5', '2.8.2', '2.7.10'] %}
+  '{{ version }}':
```

Comment:
The previous version had a `.0` at the end? Is that not the case anymore? What is reported by `Add/Remove Programs`?

Reply:
Indeed it does, thanks for the catch!